dbms/src/Common/ZooKeeper/ZooKeeperImpl.h

```diff
@@ -108,9 +108,9 @@ class ZooKeeper : public IKeeper
         const String & root_path,
         const String & auth_scheme,
         const String & auth_data,
-        Poco::Timespan session_timeout,
+        Poco::Timespan session_timeou_t,
```

Comment:
Tha't strange :)

Reply:
Agggr!
Well, I guess you can find more, after all, it was 5 hours of adding underscores.
Otherwise it is a valid name for STL class :)